controllers/mysqlcluster_controller.go

```diff
@@ -1092,6 +1092,11 @@ func (r *MySQLClusterReconciler) reconcileV1RestoreJob(ctx context.Context, req
 				Resources: []string{"pods"},
 				Verbs:     []string{"get"},
 			},
+			{
```

Comment:
How does this relate to the issue #284?

Reply:
It's nothing to do with it. I just happened to find that the restore job couldn't create events, and fixed it. Should I raise another PR?

Reply:
Yes, please.